docs/tikv-storage-configuration.md

```diff
@@ -0,0 +1,108 @@
+
+
+#### `normal-concurrency`
+
+- Specifies the thread pool size for handling normal priority requests
+- Default value: 4
+- Minimum value: 1
+
+#### `low-concurrency`
+
+- Specifies the thread pool size for handling low priority requests
+- Default value: 4
+- Minimum value: 1
+- Generally, you don’t need to ensure that the sum of high + normal + low < number of CPU cores, because a single request will be handled by only one of them
+
+#### `max-tasks-per-worker-high`
+
+- Specifies the max running operations for each thread in the read-high thread pool, which handles high priority read requests. Because a thread-pool level throttle instead of single thread level is performed, the max running operations for the read-high thread pool will be limited to `max-tasks-per-worker-high * high-concurrency`
+- Default value: 2000
+- Minimum value: 2000
+- If the number of running operations exceeds this configuration, new operations will be simply rejected without being handled and it will contain an error header telling that TiKV is busy
+- Generally, you don’t need to adjust this configuration unless you are following trustworthy advice
+
+#### `max-tasks-per-worker-normal`
+
+- Specifies the max running operations for each thread in the read-normal thread pool, which handles normal priority read requests.
+- Default value: 2000
+- Minimum value: 2000
+
+#### `max-tasks-per-worker-low`
+
+- Specifies the max running operations for each thread in the read-low thread pool, which handles low priority read requests
+- Default value: 2000
+- Minimum value: 2000
+
+#### `stack-size`
+
+- Sets the stack size for each thread in the three thread pools. For large requests, you need a large stack to handle
+- Default value: 10MB
+- Minimum value: 2MB
+
+### `[storage]`
+
+This configuration section mainly affects storage write operations, including where data is stored and the TiKV component Scheduler. Scheduler is the core component in Storage that coordinates and processes write requests. It contains a channel to coordinate requests and a thread pool to process requests.
+
+#### `data-dir`
+
+- Specifies the path to the data directory
+- Default value: /tmp/tikv/store
+- Make sure that the data directory is moved before changing this configuration
+
+#### `scheduler-notify-capacity`
+
+- Specifies the Scheduler channel size
+- Default value: 10240
+- Do not set it too small, otherwise TiKV may crash
+- Do not set it too large, because it may consume more memory 
+- Generally, you don’t need to adjust this configuration unless you are following trustworthy advice
+
+#### `scheduler-concurrency`
+
+- Specifies the number of slots of Scheduler’s latch, which controls concurrent write requests
+- Default value: 2048000
+- You can set it to a larger value to reduce latch contention if there are a lot of write requests . But it will consume more memory
```

Comment:
- Delete the extra space after "requests".
- it will consume more memory -> this consumes more memory